Kickstarter-iOS/Features/ConfirmDetails/Views/ConfirmDetailsContinueCTAView.swift

```diff
@@ -0,0 +1,163 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+public typealias ConfirmDetailsContinueCTAViewData = (
+  project: Project,
+  total: Double
+)
+
+private enum Layout {
+  enum Button {
+    static let minHeight: CGFloat = 48.0
+  }
+}
+
+protocol ConfirmDetailsContinueCTAViewDelegate: AnyObject {
+  func continueButtonTapped()
+}
+
+final class ConfirmDetailsContinueCTAView: UIView {
+  // MARK: - Properties
+
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private(set) lazy var titleAndAmountStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var titleLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var amountLabel: UILabel = { UILabel(frame: .zero) }()
+
+  private(set) lazy var continueButton: UIButton = {
+    UIButton(type: .custom)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  public weak var delegate: ConfirmDetailsContinueCTAViewDelegate?
+
+  // MARK: - Lifecycle
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+
+    self.configureSubviews()
+    self.setupConstraints()
+
+    self.continueButton.addTarget(
+      self, action: #selector(self.continueButtonTapped),
+      for: .touchUpInside
+    )
+  }
+
+  required init?(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> \.layoutMargins .~ .init(all: Styles.grid(3))
+
+    _ = self.layer
+      |> checkoutLayerCardRoundedStyle
+      |> \.backgroundColor .~ UIColor.ksr_white.cgColor
+      |> \.shadowColor .~ UIColor.ksr_black.cgColor
+      |> \.shadowOpacity .~ 0.12
+      |> \.shadowOffset .~ CGSize(width: 0, height: -1.0)
+      |> \.shadowRadius .~ CGFloat(1.0)
+      |> \.maskedCorners .~ [
+        CACornerMask.layerMaxXMinYCorner,
+        CACornerMask.layerMinXMinYCorner
+      ]
+
+    _ = self.rootStackView
+      |> verticalStackViewStyle
+      |> \.spacing .~ Styles.grid(3)
+
+    _ = self.titleAndAmountStackView
+      |> self.titleAndAmountStackViewStyle
+
+    _ = self.titleLabel
+      |> self.titleLabelStyle
+
+    _ = self.amountLabel
+      |> self.amountLabelStyle
+
+    _ = self.continueButton
+      |> greenButtonStyle
+      |> UIButton.lens.title(for: .normal) %~ { _ in Strings.Continue() }
```

Comment:
Nit/non-blocking: Mentioned this in an earlier PR but I'd prefer us to move away from Prelude code in `bindStyles` whenever we can avoid it.